src/main/java/millom/sandbox/pojos/Descriptions.java

```diff
@@ -0,0 +1,287 @@
+    this.absHumidityDescEs = descriptionsBuilder.absHumidityDescEs;
+    this.windDescEn = descriptionsBuilder.windDescEn;
+    this.windDescEs = descriptionsBuilder.windDescEs;
+    this.gtsTempDescEn = descriptionsBuilder.gtsTempDescEn;
+    this.gtsTempDescEs = descriptionsBuilder.gtsTempDescEs;
+    this.localUvIrradianceIndexDescEn = descriptionsBuilder.localUvIrradianceIndexDescEn;
+    this.localUvIrradianceIndexDescEs = descriptionsBuilder.localUvIrradianceIndexDescEs;
+    this.atmoOpacityDescEn = descriptionsBuilder.atmoOpacityDescEn;
+    this.atmoOpacityDescEs = descriptionsBuilder.atmoOpacityDescEs;
+    this.lsDescEn = descriptionsBuilder.lsDescEn;
+    this.lsDescEs = descriptionsBuilder.lsDescEs;
+    this.seasonDescEn = descriptionsBuilder.seasonDescEn;
+    this.seasonDescEs = descriptionsBuilder.seasonDescEs;
+    this.sunriseSunsetDescEn = descriptionsBuilder.sunriseSunsetDescEn;
+    this.sunriseSunsetDescEs = descriptionsBuilder.sunriseSunsetDescEs;
+  }
+
+  public String getDisclaimerEn() {
+    return disclaimerEn;
+  }
+
+  public String getDisclaimerEs() {
+    return disclaimerEs;
+  }
+
+  public String getSol_desc_en() {
+    return sol_desc_en;
+  }
+
+  public String getSol_desc_es() {
+    return sol_desc_es;
+  }
+
+  public String getTerrestrialDateDescEn() {
+    return terrestrialDateDescEn;
+  }
+
+  public String getTerrestrialDateDescEs() {
+    return terrestrialDateDescEs;
+  }
+
+  public String getTempDescEn() {
+    return tempDescEn;
+  }
+
+  public String getTempDescEs() {
+    return tempDescEs;
+  }
+
+  public String getPressureDescEn() {
+    return pressureDescEn;
+  }
+
+  public String getPressureDescEs() {
+    return pressureDescEs;
+  }
+
+  public String getAbsHumidityDescEn() {
+    return absHumidityDescEn;
+  }
+
+  public String getAbsHumidityDescEs() {
+    return absHumidityDescEs;
+  }
+
+  public String getWindDescEn() {
+    return windDescEn;
+  }
+
+  public String getWindDescEs() {
+    return windDescEs;
+  }
+
+  public String getGtsTempDescEn() {
+    return gtsTempDescEn;
+  }
+
+  public String getGtsTempDescEs() {
+    return gtsTempDescEs;
+  }
+
+  public String getLocalUvIrradianceIndexDescEn() {
+    return localUvIrradianceIndexDescEn;
+  }
+
+  public String getLocalUvIrradianceIndexDescEs() {
+    return localUvIrradianceIndexDescEs;
+  }
+
+  public String getAtmoOpacityDescEn() {
+    return atmoOpacityDescEn;
+  }
+
+  public String getAtmoOpacityDescEs() {
+    return atmoOpacityDescEs;
+  }
+
+  public String getLsDescEn() {
+    return lsDescEn;
+  }
+
+  public String getLsDescEs() {
+    return lsDescEs;
+  }
+
+  public String getSeasonDescEn() {
+    return seasonDescEn;
+  }
+
+  public String getSeasonDescEs() {
+    return seasonDescEs;
+  }
+
+  public String getSunriseSunsetDescEn() {
+    return sunriseSunsetDescEn;
+  }
+
+  public String getSunriseSunsetDescEs() {
+    return sunriseSunsetDescEs;
+  }
+  public static class DescriptionsBuilder{
```

Comment:
If I am not wrong this builder is used only in a unit test? If that is the case and because the `Description` 
constructor is public, then it could be an idea to just remove the `DescriptionBuilder` class and in stead call
the `Description` constructor directly from the unit test? But this kind of depends on what the plans are later
on I think?